webcam/public/index.html

```diff
@@ -0,0 +1,77 @@
+
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+  </head>
+  <body>
+    Name:<input type="text" id="Name" />
+    Comment:<input type="text" id="Comment" />
+  <!--<input type="submit" id="button" />-->
+    Picture:<input type="submit" value="take picture" id = "click"/>
+      <pre id="chatid"/>
+      <video id = "v1"></video>
+      <canvas width = 400 height = 400 id ='c1'></canvas>
+
+ <script>
+/*const button =document.getElementById("button");
+button.onclick=()=>{
+  const name=document.getElementById("Name").value;
+  const comment=document.getElementById("Comment").value;
+  const xhttp=new XMLHttpRequest();
+  xhttp.open('POST', '/newchat');
+  xhttp.setRequestHeader('content-type', 'application/json');
+  xhttp.send(JSON.stringify(
+        {name, comment}
+  )
+  );
+}
+const startchat=()=>{
```

Comment:
why is chatbox commented out?

Reply:
lesson 8 tells you to only "Get access to webcam. When user clicks a button, take a picture by drawing it to canvas" i dont think so chatbox is required in this task